app/dashboard/templates/extend_issue_deadline.html

```diff
@@ -0,0 +1,74 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<div id="extend_deadline" class="white-light-bg">
+  <div class="row">
+    <div class="col-12 closebtn">
+      <a rel="modal:close" href="javascript:void" class="close" aria-label="Close dialog">
+        <span aria-hidden="true">&times;</span>
+      </a>
+    </div>
+    <div class="col-12">
+      <h5 class="text-center font-title">{% trans "Extend Issue Deadline" %}</h5>
+    </div>
+    {% if user_logged_in %}
+      <div class="col-12">
+        <h5 class="text-center font-body">{% trans "Alpha Seed Implementation (New Feature)" %}</h5>
+        <form class="font-body">
+          <div>
+            <input type="hidden" name="currentExpire" value="{{ bounty.expires_date }}" />
+            <div id="deadline-section" class="pt-2">
+                <label class="form__label" for="expirationTimeDelta">{% trans "Extend by:" %}</label>
+                <div class="form__select2">
+                    <select class="js-select2" name="expirationTimeDelta">
```

Comment:
Can we adjust the rest of the indentation throughout this file to use 2 space indentation?